md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/PropertiesAPI.java

```diff
@@ -56,12 +58,14 @@
 
     @ResponseBody
     public RestWrapper delete(@PathVariable("id") Integer processId, Principal principal) {
-
         RestWrapper restWrapper = null;
         try {
-
-
-            com.wipro.ats.bdre.md.dao.jpa.Process process = new Process();
+            Process parentProcess=processDAO.get(processId);
+            if (parentProcess.getProcess()!=null)
+                  processDAO.securityCheck(parentProcess.getProcess().getProcessId(),principal.getName(),"write");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "write" 10 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1192)